### PR TITLE
some updates to the Install documentation

### PR DESCRIPTION
**Pull request by tobiasd**
I did a test installation the other day on a Debian stable box and everything worked fine. However I stumbled over some things in the documentation for the installation.

### CODE REVIEW

File: doc/Install.md [hunk 1]
```diff
@@ -47,7 +47,6 @@ For alternative server configurations (such as Nginx server and MariaDB database
 ### Optional 
 
 * PHP ImageMagick extension (php-imagick) for animated GIF support.
-* [Composer](https://getcomposer.org/) for a git install
```
Comment (MrPetovan) [inline doc/Install.md:50]: Why remove this line?

Comment (tobiasd) [reply doc/Install.md:50]: Composer is shipped in the bin directory isn't it?

Comment (MrPetovan) [reply doc/Install.md:50]: Yes, sorry, I forgot about that.

Comment (tobiasd) [reply doc/Install.md:50]: Ok *phew*, your question got me thinking if the `composer.phar` in the directory was only a part of the composer game but the other part had to be installed on the system. And you do know the composer ecosystem better then I do.

**Pull request merged**